h2o-hive/src/main/java/water/hive/DelegationTokenRefresher.java

```diff
@@ -39,10 +44,10 @@ public static void setup(Configuration conf, String iceRoot) throws IOException
   }
   
   private static String writeKeytabToFile(String authKeytab, String iceRoot) throws IOException {
-    h2omapper.makeSureIceRootExists(iceRoot);
```

Comment:
Is this correct in Sparkling Water's context? Or maybe - is this correct naming in a general context?